alex-ansible-playbook.yml

```diff
@@ -0,0 +1,73 @@
+- name: Install the To-do app
+  hosts: webservers
+  remote_user: ec2-user
+
+  tasks:
+    - name: Install git
+      ansible.builtin.yum:
+        name:
+          - git
+        state: present
+      become: yes
+
+    - name: Install python3
+      ansible.builtin.yum:
+        name: python3
+        state: latest
+
+    - name: Intall poetry
+      ansible.builtin.shell:
+        cmd:
+          curl -sSL
+          https://install.python-poetry.org |
+          python3 -
+
+    - name: Create Directory for To Do app
+      ansible.builtin.file:
+        path: /opt/todoapp
+        owner: ec2-user
+        state: directory
+      become: yes
+
+    - name:
+        Get the latest version of the code from
+        Git
+      ansible.builtin.git:
+        repo: https://github.com/talexandru1987/DevOps-Course-Starter.git
+        dest: /opt/todoapp
+
+    - name: Install project dependencies
+      ansible.builtin.shell:
+        cmd:
+          cd /opt/todoapp;
+          /home/ec2-user/.local/bin/poetry install
+      become: yes
+
+- name: Install .env from template
+  hosts: webservers
+  vars_prompt:
+    - name: key
+      prompt: "Enter the trello key"
+      private: no
```

Comment:
Minor here because of how we're running the infrastructure, but worth considering whether we should mark these private by default as a future-proofing exercise; if someone were to later move this into a CD pipeline for example, it's plausible that we accidentally leak our token into the pipeline logs, which we'd definitely want to avoid!